src/main/scala/com/netflix/edda/mongo/MongoDatastore.scala

```diff
@@ -69,18 +69,27 @@ object MongoDatastore {
     import collection.JavaConverters._
     obj match {
       case o: BasicDBObject => {
-        o.keySet.asScala.map((key: String) => (key -> mongoToScala(o.get(key)))).toMap
+        o.keySet.asScala.map((key: String) => (mongoDecodeString(key) -> mongoToScala(o.get(key)))).toMap
       }
       case o: BasicDBList => {
         List.empty[Any] ++ o.asScala.map(mongoToScala(_))
       }
       case o: Date => new DateTime(o)
+      case o: String => mongoDecodeString(o)
       case o: AnyRef => o
       case null => null
       case other => throw new java.lang.RuntimeException("mongoToScala: don't know how to handle: " + other)
     }
   }
 
+  def mongoDecodeString(str: String): String = {
+    str.replace("\\0xFF0E", ".").replace("0xFF0E", ".")
```

Comment:
Few questions:

* Shouldn't this be `0x002E`?
* Perhaps the unicode escaping for [json](http://www.json.org/) would be more easier to work with if accessing the mongo data from some other tool. That is `\uFF0E` instead of `\0xFF0E`. Thoughts?
* The `$` is also special in mongo, do you think we should go ahead and escape it as well?